libqtile/widget/camcheck.py

```diff
@@ -0,0 +1,43 @@
+# A qtile widget written to show active indication for mic or camera
+# written by bhauvk sharma (https://github.com/desdemona2)
+
+from libqtile.widget import base
+import os
+
+
+class active(base.ThreadPoolText):
```

Comment:
This widget needs a better name and it should be capitalised. You also need to add the widget to the `widgets` dict in `libqtile/widget/__init__.py`.